src/sqlfluff/cli/formatters.py

```diff
@@ -14,6 +15,7 @@
     wrap_field,
 )
 from sqlfluff.cli.outputstream import OutputStream
+
```

Comment:
Nitpick: Why was this blank line added?

Reply:
Seperates the `cli` imports from the `core` imports. Seemed like a good idea at the time 🤷

Reply:
No worries!!